app/controllers/customers_controller.rb

```diff
@@ -0,0 +1,61 @@
+class CustomersController < ApplicationController
+  layout 'settings'
+  before_action :set_customer, only: [:edit, :update, :destroy]
+  before_action :require_organization
+
+  def new
+    @customer = current_organization.customers.build
+  end
+
+  def edit
+  end
+
+  def index
+    @customers = current_organization.customers
+
+    respond_to do |format|
+      format.html
+      format.json { render json: @customers , only: [:id, :name] }
+    end
+  end
+
+  def create
+    @customer = current_organization.customers.build(customer_params)
+
+    if @customer.save
+      redirect_to customers_path, notice: 'Customer was successfully created.'
+    else
+      render action: 'new'
+    end
+  end
+
+  def autocomplete
+    @customers = current_organization.customers.pluck(:id, :name)
```

Comment:
@Gataniel there will be text that user has entered in select2. So you should select all customers which name starts with this term. Or include. So it be something like `where("name like ?", "%term%")`.